### ISSUE TITLE
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted

### ISSUE THREAD

**Issue by lumenwrites** [logtail/logtail-js#20]
Hi! I'm trying to set up this library with my Next.js app.

Here's my `logger.ts` file, which I'm planning to import in my app to log events:
```javascript
import { Browser as LogtailBrowser } from "@logtail/js";
import { Node as LogtailNode } from "@logtail/js"

const logClient = new LogtailBrowser(process.env.NEXT_PUBLIC_LOGTAIL_TOKEN)
const logServer = new LogtailNode(process.env.NEXT_PUBLIC_LOGTAIL_TOKEN)

export { logClient, logServer }
``` 

Now when I'm running my app, I am getting a warning:
```
warn  - ./node_modules/@logtail/node/dist/es6/context.js
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted

Import trace for requested module:
./node_modules/@logtail/node/dist/es6/node.js
./node_modules/@logtail/node/dist/es6/index.js
./node_modules/@logtail/js/dist/es6/index.js
./logger/logger.ts
```

Can you tell me what that means, and how do I fix it?

**Comment by adikus** [comment]
Hi @lumenwrites! Thanks for reaching out.

It looks like you are inadvertently requiring the server side node Logtail package from your client side code. Could you try to split up your `logger.ts` such that in only contains imports either for server (node), or client (browser) Logtail packages? That way your client side code shouldn't attempt to load the package meant for node.

**Comment by DanielKAtBrilliant** [comment]
Hi, I see the same issue even if I just use the Browser (client) logger for client-side code code:

```
import { Browser as Logtail } from '@logtail/js';
export const logger = new Logtail(process.env.NEXT_PUBLIC_LOGTAIL_TOKEN);
```

```
warn  - ./node_modules/@logtail/node/dist/es6/context.js
Critical dependency: require function is used in a way in which dependencies cannot be statically extracted
```

**Comment by curusarn** [comment]
Thank you, everyone, for reporting this.
We're tracking this issue, and we'll make sure to keep you posted.

**Comment by curusarn** [comment]
Hi everyone,

I have just released [v0.3.0](https://github.com/logtail/logtail-js/releases/tag/v0.3.0) which shouldn't give you warnings anymore. 
Update to `v0.3.0` and let me know if the issue persists.

You can always message us at [hello@betterstack.com](mailto:hello@betterstack.com). We're happy to help. 🙏
